pupil-spa/src/app/question/question.component.spec.ts

```diff
@@ -18,7 +25,9 @@ describe('QuestionComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
     // prevent Timeout's being set
-    spyOn(component, 'ngAfterViewInit').and.returnValue(null);
+    // NOTE - calling this makes no difference to sendTimeoutEvent test.
+    // all other tests work without the call below.
+    //spyOn(component, 'ngAfterViewInit').and.returnValue(null);
```

Comment:
@jon-shipley i checked every test with a false=true call and none of them failed without it, apart from the audit ones, which rely on ngAfterViewInit